lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -621,17 +621,7 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
         parameters[:discount_amount] = options[:discount_amount] if options[:discount_amount]
         parameters[:ships_from_postal_code] = options[:ships_from_postal_code] if options[:ships_from_postal_code]
 
-        if options[:line_items]
-          items = []
```

Comment:
Is it possible that `line_items` ever comes in as something other than an array? That's really the only reason I could see for keeping this. If not, 👍

Reply:
It's probably unlikely, but if it's not, it will cause a gateway error rather than an AM error, so I'm ok with that.

Reply:
Most other adapters follow this pattern instead.